src/analysis/current_pace.rs

```diff
@@ -7,7 +7,7 @@ use crate::{analysis, timing::Snapshot, TimeSpan, TimerPhase};
 /// Calculates the current pace of the active attempt based on the comparison
 /// provided. If there's no active attempt, the final time of the comparison is
 /// returned instead.
```

Comment:
Should we also have something like this here? "Additionally a boolean is returned that indicates if the value is currently actively changing as time is being lost."

Reply:
Yeah that should've probably mentioned it. I'll try to sneak it into some future commit.